# Correlating RNAseq profiles with cellular phenotypes

Given an RNAseq gene expression profile of a cell, can we determine what phenotype (e.g. cell growth vs. stasis) that cell has?


## Related work

- [Signatures of cell death and proliferation in perturbation transcriptomics dataâ€”from confounding factor to effective prediction | Nucleic Acids Research | Oxford Academic](https://academic.oup.com/nar/article/47/19/10010/5573547#165684176)
  - [bence-szalai/Cell-death-signatures](https://github.com/bence-szalai/Cell-death-signatures)
- [Predicting tumor response to drugs based on gene-expression biomarkers of sensitivity learned from cancer cell lines | BMC Genomics](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-07581-7#Sec2)
- [Common and cell-type specific responses to anti-cancer drugs revealed by high throughput transcript profiling | Nature Communications](https://www.nature.com/articles/s41467-017-01383-w#Sec7)
  - no usable data; uses clustering approach

